packages/i18nify-js/src/modules/currency/__tests__/getCurrencyList.test.ts

```diff
@@ -17,10 +19,12 @@ describe('getCurrencyList', () => {
   it("check the values of 'symbol' and 'name' properties for a sample currency", () => {
     const currencyList = getCurrencyList();
     const sampleCurrencyCode = 'USD';
-    const sampleCurrency = currencyList[sampleCurrencyCode];
+    const sampleCurrency: Currency[string] = currencyList[sampleCurrencyCode];
```

Comment:
doesn't this translate to `string` ? 
Also, is it needed ?

Reply:
yes, it's needed, since currency interface looks like below:

```javascript
export interface Currency {
  [key: string]: {
    name: string;
    minor_unit: string;
    symbol: string;
  };
}
```

and sampleCurrency is supposed to store value like below:

```javascript
{
    name: "Indian Rupee",
    minor_unit: "2",
    symbol: "₹"
  }
```